### PR TITLE
Use padd.sh URLs

### PR DESCRIPTION
**Pull request by dschaper**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [ ] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Move to `https://install.padd.sh` URL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +42,15 @@ cd ~ ; echo "if [ \"\$TERM\" == \"linux\" ] ; then\n  while :\n  do\n    ./padd.
...
+```bash
+cd ~
+curl -sSL https://install.padd.sh -o padd.sh
 ```
-
-**Note: if you are already running Chronometer2 v1.3.1 or below, you’ll need to follow [these instructions](https://github.com/jpmck/PADD/wiki/Updating-from-Chronometer2)!**
 
 ## FAQ
-*Answers to frequently asked questions can be found in this repo’s [wiki](https://github.com/jpmck/PADD/wiki/FAQ).*
+*Answers to frequently asked questions can be found in this repo’s [wiki](https://github.com/pi-hole/PADD/wiki/FAQ).*
```
Comment (yubiuser) [inline README.md:56]: We don't have a FAQ....

**Review by yubiuser — CHANGES_REQUESTED** [review]
We should update, clean and fix the wiki before merging this to master

**Review by yubiuser — CHANGES_REQUESTED** [review]
L7: The image does not exist anymore. 

Additionally, before we merge this, we need to update the wiki as well (contains old links...)

**Comment by rdwebdesign** [comment]
Should we reopen this PR?

**Comment by dschaper** [comment]
That's up to you all, the domain is there and available to use.

**Comment by rdwebdesign** [comment]
I will reopen the PR. 

My opinion is: we should use the domain.
As always, I want to hear the opinion of the other team members.

**Comment by rdwebdesign** [comment]
The proposed change on this PR is simple, but there are a lot of conflicts here.

**Review by yubiuser — APPROVED** [review]

**Pull request merged**